content/documentation/staging/tutorial/06-secure/01-traffic-policies.adoc

```diff
@@ -0,0 +1,128 @@
+---
+title: Authorization Policies
+weight: 56
+---
+:icons: font
+:sectlinks:
+
+https://istio.io/latest/docs/concepts/security/[Security, window="_blank"] is one of the main pillars of Istio features.
+
+The Istio https://istio.io/latest/docs/concepts/security/#high-level-architecture[Security High Level Architecture] provides a comprehensive solution to design and implement multiple security scenarios.
+
+In this tutorial we will show how Kiali can use telemetry information to create security policies for the workloads deployed in a given namespace.
+
+Istio telemetry aggregates the ServiceAccount information used in the workloads communication. This information can be used to define authorization policies that deny and allow actions on future live traffic communication status.
+
+This step will show how we can define authorization policies for the *travel-agency* namespace in the Travel Demo application, for all existing traffic in a given time period.
+
+Once authorization policies are defined, a new workload will be rejected if it doesn't match the security rules defined.
+
+NOTE: *Step 1:* Undeploy the *loadtester* workload from *travel-portal* namespace
+
+In this example we will use the *loadtester* workload as the "intruder" in our security rules.
+
+If we have followed the previous tutorial steps, we need to undeploy it from the system.
+
+[source,bash]
+----
+kubectl apply -f <(curl -L https://raw.githubusercontent.com/kiali/demos/master/travels/travel_loadtester.yaml) -n travel-portal
+----
+
+We should validate that telemetry has updated the *travel-portal* namespace and "Security" can be enabled in the Graph Display options.
+
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/06-01-travel-portal-graph.png" title="Travel Portal Graph">
+    <img src="/images/tutorial/06-01-travel-portal-graph.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+NOTE: *Step 2:* Create Authorization Policies for current traffic for *travel-agency* namespace
+
+Every workload in the cluster uses a https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/[Service Account, window="_blank"].
+
+*travels.uk*, *viaggi.it* and *voyages.fr* workloads use the default *cluster.local/ns/travel-portal/sa/default" ServiceAccount defined automatically per namespace.
```

Comment:
I think `*` is meant to be a wildcard here. So I think this is just missing the left-quote:
```suggestion
*travels.uk*, *viaggi.it* and *voyages.fr* workloads use the default "*cluster.local/ns/travel-portal/sa/default" ServiceAccount defined automatically per namespace.
```